www/master/components/dashboard/test/controllers/header.spec.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+describe("header controller", function() {
```

Comment:
I've updated the test descriptions to be a bit more helpful and added some more tests. If you run karma with the `story` reporter, you'll get some output that should help describe what the code under test is expected to do:

    $ cd www/master
    $ karma start --browsers PhantomJS --reporters story
    ...
    PhantomJS 1.9.8 (Mac OS X) - header controller:
    PhantomJS 1.9.8 (Mac OS X) 	- subPages:
    PhantomJS 1.9.8 (Mac OS X) 		is defined PASSED
    PhantomJS 1.9.8 (Mac OS X) 		is an array PASSED
    PhantomJS 1.9.8 (Mac OS X) 		is not empty PASSED
    PhantomJS 1.9.8 (Mac OS X) 
    PhantomJS 1.9.8 (Mac OS X) 		- each subPage:
    PhantomJS 1.9.8 (Mac OS X) 			has a category PASSED
    PhantomJS 1.9.8 (Mac OS X) 			has a name PASSED
    PhantomJS 1.9.8 (Mac OS X) 			has a value PASSED
    PhantomJS 1.9.8 (Mac OS X) 
    PhantomJS 1.9.8 (Mac OS X) 	- Pages:
    PhantomJS 1.9.8 (Mac OS X) 		does not change location on first detected change PASSED
    PhantomJS 1.9.8 (Mac OS X) 		changes location on second detected change PASSED

I hope this helps!